src/components/ProgramList.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from "react";
+import {
+    Box,
+    Heading,
+    Wrap,
+    WrapItem,
+    Badge,
+    Center,
+    AspectRatio,
+    Spacer,
+    Image,
+} from "@chakra-ui/react";
+
+export const ProgramList: React.FC = () => {
+    const handleClick = (event) => {
+        event.preventDefault();
+        console.log(event.target);
```

Comment:
Remove log

Reply:
i put a todo, this is a proof of concept of onclick function, it will be replaced by the link to the program details page later